CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 23.06.1+ (???)
 ------------------------------------------------------------------------
+- Feature: [#7627] Allow mouse scrolling on +/- stepper widgets to change their values.
```

Comment:
Not sure how this turned into #7627 instead of #2019 without anyone noticing, myself included...

Reply:
😄 We are all guilty